components/brave_news/browser/brave_news_p3a.cc

```diff
@@ -165,7 +164,7 @@ void NewsMetrics::RecordTotalActionCount(ActionType action,
                                          kSidebarFilterUsageBuckets, total);
       break;
     default:
```

Comment:
Is this a case here and in other similar places that there are no other values, and the switch is already covering all cases? if that's so, shouldn't we remove also the `default:` entry, so the compiler can alert us if a new key gets added to the switch?

Reply:
yes, i think that's better. changed where applicable.